packages/fast-components-react-base/src/breadcrumb/breadcrumb.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, mount, shallow } from "enzyme";
+import Breadcrumb, {
+    BreadcrumbClassNameContract,
+    BreadcrumbUnhandledProps,
+} from "./breadcrumb";
+import { KeyCodes } from "@microsoft/fast-web-utilities";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+const managedClasses: BreadcrumbClassNameContract = {
+    breadcrumb: "breadcrumb-class",
+    breadcrumb_item: "breadcrumb-item-class",
+    breadcrumb_item__current: "breadcrumb-item-current-class",
+    breadcrumb_ol: "breadcrumb-ol-class",
+    breadcrumb_separator: "breadcrumb-separator-class",
+};
+
+describe("breadcrumb", (): void => {
+    test("should have a displayName that matches the component name", () => {
+        expect((Breadcrumb as any).name).toBe(Breadcrumb.displayName);
+    });
+
+    test("should not throw if managedClasses are not provided", () => {
+        expect(() => {
+            shallow(<Breadcrumb />);
+        }).not.toThrow();
+    });
+
+    test("should implement unhandledProps", (): void => {
+        const unhandledProps: BreadcrumbUnhandledProps = {
+            "aria-controls": "controls",
+        };
+
+        const rendered: any = shallow(<Breadcrumb {...unhandledProps} />);
+
+        expect(rendered.first().prop("aria-controls")).toEqual("controls");
+    });
+
+    test("should set current class name on last rendered", (): void => {
+        const rendered: any = shallow(
+            <Breadcrumb managedClasses={managedClasses}>
+                <a />
+                <a />
+                <a />
+                <a key="blah" />
+            </Breadcrumb>
+        );
+
+        /* tslint:disable-next-line */
+        console.log(rendered.instance().props.children[3].props.class);

```

Comment:
I don't think we should have a console.log in our tests.

Reply:
This is just a product of me testing something at home. But didn't want to loose my progress so I pushed my changes. This is now longer part of the review.